brayns/engine/material/Material.cpp

```diff
@@ -35,6 +50,7 @@ Material &Material::operator=(const Material &other)
     _handle = ospray::cpp::Material("", _handleName);
     _data = other._data->clone();
     _data->pushTo(_handle);
+    _flag = true;
```

Comment:
This looks a bit weird, I would do something like _flag.setModified(true).

Reply:
Will change on next PR